### ISSUE TITLE
After installing SDK 9.0 RC2, creating ASP.NET Core Web App (Razor Pages) (Individual) project, some packages are not loaded successfully.

### ISSUE THREAD

**Issue by vmykagapuz** [dotnet/sdk#43341]
**REGRESSION**
1. Worked in SDK RC1

**INSTALL STEPS**

1. Clean machine: Win11 x64 22h2 ENU
3. Install SDK 9.0 RC2 daily build
4. Apply 9.0 Feed 
5. Install VSCode or VS

**REPRO STEPS**
1. Open CMD, create an ASP.NET Core Web App (Razor Pages) using following commands:
`dotnet new razor -au individual -o webapp1`
2. Open this project using VSCode or VS.

**ACTUAL**
Dependencies | Packages show warning icon. Some packages are not loaded successfully.
![image](https://github.com/user-attachments/assets/c0106145-4862-4a58-9061-eea0b53ed071)

**EXPECTED**
All packages should be loaded successfully.

**Comment by MackinnonBuck** [comment]
@wtgodbe, it looks like the templates are referencing a package that can't get resolved. Do you know what might cause this? What decides the version of a package in the generated project?

**Comment by wtgodbe** [comment]
What feed are you using? I traced back the build, and those packages should definitely be available in the dotnet9 feed

**Comment by v-elenafeng** [comment]
> What feed are you using? I traced back the build, and those packages should definitely be available in the dotnet9 feed

Yes, we used the [dotnet9 feed](https://github.com/dotnet/sdk?tab=readme-ov-file#for-net-9-builds).

**Comment by mkArtakMSFT** [comment]
EF issue should be fixed now. @AndriySvyryd can you please link the PR here? Thanks!

**Comment by mkArtakMSFT** [comment]
What are the actual warnings / errors shown in the console?

**Comment by mkArtakMSFT** [comment]
@vmykagapuz does the project run or not after F5?>

**Comment by AndriySvyryd** [comment]
@mkArtakMSFT I think this is a feed/coherency issue

**Comment by vmykagapuz** [comment]
> What are the actual warnings / errors shown in the console?

There is no warning/error message in the console, this issue occurs right after creating the project without running it.

**Comment by vmykagapuz** [comment]
> @vmykagapuz does the project run or not after F5?>

The warning sign does not affect the running of the project, and here is the warning message shown in the error list.
![image](https://github.com/user-attachments/assets/24613d5c-2a53-445a-acc6-549a7e545115)

**Comment by v-elenafeng** [comment]
As Jose discovered [here](https://github.com/dotnet/aspire/issues/5679#issuecomment-2347134691), the issue is caused by the new feature in NuGet to surface vulnerabilities in dependencies. And the warnings can be resolved by manually updating the vulnerable packages. There is an issue https://github.com/dotnet/AspNetCore-ManualTests/issues/2929#issuecomment-2348161898 tracking the vulnerable packages.

```
Project 'WebApplication6' has the following dependency graph(s) for 'System.Text.Json':

  [net9.0]
   │
   ├─ Microsoft.EntityFrameworkCore.SqlServer (v9.0.0-rc.2.24460.3)
   │  └─ Microsoft.Data.SqlClient (v5.1.6)
   │     ├─ Azure.Identity (v1.11.4)
   │     │  ├─ Azure.Core (v1.38.0)
   │     │  │  ├─ System.ClientModel (v1.0.0)
   │     │  │  │  ├─ System.Memory.Data (v1.0.2)
   │     │  │  │  │  └─ System.Text.Json (v7.0.3)
   │     │  │  │  └─ System.Text.Json (v7.0.3)
   │     │  │  ├─ System.Memory.Data (v1.0.2)
   │     │  │  │  └─ System.Text.Json (v7.0.3)
   │     │  │  └─ System.Text.Json (v7.0.3)
   │     │  └─ System.Text.Json (v7.0.3)
   │     ├─ Microsoft.IdentityModel.JsonWebTokens (v6.35.0)
   │     │  └─ System.Text.Json (v7.0.3)
   │     └─ Microsoft.IdentityModel.Protocols.OpenIdConnect (v6.35.0)
   │        └─ System.IdentityModel.Tokens.Jwt (v6.35.0)
   │           └─ Microsoft.IdentityModel.JsonWebTokens (v6.35.0)
   │              └─ System.Text.Json (v7.0.3)
   └─ Microsoft.EntityFrameworkCore.Tools (v9.0.0-rc.2.24460.3)
      └─ Microsoft.EntityFrameworkCore.Design (v9.0.0-rc.2.24460.3)
         └─ Microsoft.CodeAnalysis.Workspaces.MSBuild (v4.8.0)
            └─ System.Text.Json (v7.0.3)
```

**Comment by AndriySvyryd** [comment]
Then it will be fixed by https://github.com/dotnet/efcore/pull/34666

**Comment by v-elenafeng** [comment]
Verified that this issue has been fixed in VS 17.12 P3 (including 9.0 rc2).
